src/fundus/utils/more_async.py

```diff
@@ -33,7 +33,7 @@ async def async_next(iterator: AsyncIterator[_T], default: Union[_VT, _Sentinel]
             raise StopAsyncIteration
 
 
-async def batched_interleave_longest(*iterators: AsyncIterator[_T]) -> AsyncIterator[Iterable[_T]]:
+async def zip_longest(*iterators: AsyncIterator[_T]) -> AsyncIterator[Tuple[_T, ...]]:
```

Comment:
I changed the code to use `aioitertools` zip_longest instead.

Reply:
Great. But the `raise result` part from before is not entirely gone. Was it just leftover code?

Reply:
Actually the whole function is gone or am i missing something?

Reply:
Yes, but not all of the behaviour of the original function has been translated to the new version. I don't know the purpose of the "raise" part so I was just asking.

Reply:
> but not all of the behaviour of the original function has been translated to the new version

It is. aioitertools.zip_longest handles the raise part.

Reply:
Oh yes, I missed that, you are right. So the reason for the raise part is because the authors use `return_exceptions=True` for their `gather` call. https://github.com/omnilib/aioitertools/blob/53e15e38c4e1b5fea4876fa5f35883f11de015ff/aioitertools/itertools.py#L523